src/endpoints/v1.es6.js

```diff
@@ -658,6 +690,38 @@ class APIv1Endpoint {
           defer.reject('Comment', options.model, valid);
           return defer.promise;
         }
+      },
+
+      patch: function(options={}) {
+        var uri = options.origin + '/api/editusertext';
+
+        if (!options.model) {
+          throw new NoModelError('/api/editusertext');
```

Comment:
in the constructor for noModelError the variable seems to imply the endpoint is what should be supplied